executor.py

```diff
@@ -26,6 +27,8 @@ def __init__(
 
         self._match_args = match_args or {}
         self._storage = DocumentArrayMemmap(self.workspace)
+        self._embedding_shape = None
+        self.logger = JinaLogger('simple_indexer')
```

Comment:
```suggestion
        self.logger = JinaLogger(self.metas.name)
```

Reply:
Addressed, please see [this](https://github.com/jina-ai/executor-simpleindexer/pull/21/commits/a2a78c5be290e6d6d2f6f89b0ac117664084ed3f) commit for the change.